app/styles/play/play-level-view.sass

```diff
@@ -289,8 +289,11 @@ $level-resize-transition-time: 0.5s
     .game-container, .level-content, #game-area, #canvas-wrapper
       height: 100%
 
+    #canvas-wrapper
```

Comment:
Remove #canvas-wrapper from above declaration if we are going to immediately overwrite it here? `calc` is supported in all browsers we target so don't need a fallback.